packages/manager/modules/pci/src/projects/project/notebooks/notebooks.service.js

```diff
@@ -9,6 +9,7 @@ export default class NotebookService {
       headers: {
         'X-Pagination-Mode': 'CachedObjectList-Pages',
         'X-Pagination-Size': 50000,
```

Comment:
Aren't we using pagination ? Maybe it could be better than fetching all

Reply:
We checked with the PU and they said that the max rows does not exceed 50... The general use-case is just around 10 rows...
Hence we decided to keep it simple and fetch it at 1 shot